OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -14,15 +14,18 @@ struct AServer
 	var string IP;
 	var bool Locked;
 	var string GameMode;
+	var int iMaxPlayers,iNumPlayers;//1.5 - max players/current players

```

Comment:
Adds additional fields to AServer struct, so we can use array<AServer> ServerList to hold all our info.  Instead of putting received server info directly into the menu list items.